GVFS/GVFS.Build/GVFS.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup Label="Parameters">
     <GVFSVersion>0.2.173.2</GVFSVersion>
-    <GitPackageVersion>2.20180814.4</GitPackageVersion>
+    <GitPackageVersion>2.20181009.1-pr</GitPackageVersion>

```

Comment:
What was the fix in Git?

Reply:
See https://github.com/Microsoft/git/pull/33

A folder entry in the modified paths was not being used as recursive in the case of untracked files.  So if there was the entry foo/ in the modified paths, git was not going into any of the folders of foo to look for untracked files.

This had to do with the fact that while using the exclude we were "unexcluding" folders so if we had the file foo/bar/test.txt, we would need the !foo/, !foo/bar/ in order for git to traverse the directories and find the untracked file.  We still do that but because we no longer add to the modified paths if there is a parent directory entry, git needs to treat the folder entry as recursive.